### PR TITLE
CF/BF/INAV - Delete unused 'angleAtMin' and 'angleAtMax'.

### PR DESCRIPTION
**Pull request by Linjieqiang**
catch up with cleanflight https://github.com/cleanflight/cleanflight/pull/2676

### CODE REVIEW

**Comment by Linjieqiang** [comment]
@martinbudden Can you help to review this code ?

File: src/main/fc/fc_msp.c [hunk 1]
```diff
@@ -598,8 +598,6 @@ static bool mspFcProcessOutCommand(uint8_t cmdMSP, sbuf_t *dst, mspPostProcessFn
             sbufWriteU16(dst, servoParams(i)->max);
             sbufWriteU16(dst, servoParams(i)->middle);
             sbufWriteU8(dst, servoParams(i)->rate);
-            sbufWriteU8(dst, servoParams(i)->angleAtMin);
-            sbufWriteU8(dst, servoParams(i)->angleAtMax);
```
Comment (martinbudden) [inline src/main/fc/fc_msp.c:602]: Removing these lines will break MSP compatibility. Replace each with `sbufWriteU8(dst, 0);` instead.

File: src/main/fc/fc_msp.c [hunk 2]
```diff
@@ -1473,8 +1471,6 @@ static mspResult_e mspFcProcessInCommand(uint8_t cmdMSP, sbuf_t *src)
             servoParamsMutable(i)->max = sbufReadU16(src);
             servoParamsMutable(i)->middle = sbufReadU16(src);
             servoParamsMutable(i)->rate = sbufReadU8(src);
-            servoParamsMutable(i)->angleAtMin = sbufReadU8(src);
-            servoParamsMutable(i)->angleAtMax = sbufReadU8(src);
```
Comment (martinbudden) [inline src/main/fc/fc_msp.c:1477]: Removing these lines will break MSP compatibility. Replace each with `sbufReadU8(src);` instead.

**Review by martinbudden — CHANGES_REQUESTED** [review]
Please make changes as per comments.

**Comment by Linjieqiang** [comment]
@martinbudden Maybe thinking about to change MSP protocol in INAV configuration project?

**Comment by Linjieqiang** [comment]
OK.I will take a pr again about this problem.See this:https://github.com/iNavFlight/inav/pull/1453

**Pull request closed without merging**